python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -880,6 +883,25 @@ keys:
                       max: 65535
                       convert_types:
                         - str
+            aaa:
+              type: dict
+              keys:
+                unresponsive:
+                  type: dict
+                  description: Configure AAA timeout options.
+                  keys:
+                    eap_response:
+                      type: str
+                      description: EAP response to send.
+                      valid_values:
+                        - "success"
+                        - "disabled"
+                    action:
+                      type: dict
```

Comment:
did you re-verify the timeout parameter in other EOS version?

Reply:
@MaheshGSLAB tested this PR on lab.